www/docusaurus.config.js

```diff
@@ -39,24 +39,13 @@ module.exports = {
         {
           to: 'docs',
           label: 'Docs',
+          position: 'right',
           activeBaseRegex: 'docs(/?)$',
         },
         {
           to: 'docs/quickstart',
-          label: 'Quickstart',
-        },
-        {
-          to: 'docs/awesome-trpc',
-          label: 'Awesome tRPC Collection',
-        },
-        {
-          to: 'docs/nextjs',
-          label: 'Usage with Next.js',
-        },
```

Comment:
Hmm why do you wanna delete these?

Reply:
It's based on the new proposed design
<img width="1373" alt="CleanShot 2022-07-21 at 13 36 27@2x" src="https://user-images.githubusercontent.com/20271968/180194297-59b86dd9-59f0-4d20-a360-8e7655ee0e3e.png">

The two deleted links are also in the first section of the docs immediately, if necessary we could probably have a dropdown button that would link some important sections of the docs because now that the navbar has the container class of tailwind it won't fit many items. As for the twitter and discord links we will have those underneath the main introductory content in the landing page so they aren't gone they will just be moved.